### PR TITLE
UI basics added, token generator added and views added

### PR DESCRIPTION
**Pull request by corez92**
Built upon what Adrian's initial template PR, I changed the templates a little and added some views along with the token logic. Of course this was also me experimenting with different stuff, so bear with me if it's "simple".

### CODE REVIEW

**Comment by Mester** [comment]
Would be great with screenshots of how it looks.

**Comment by corez92** [comment]
Screenshots: http://imgur.com/a/GTXCH

File: tele_giphy/game/templates/game/index.html [hunk 1]
```diff
@@ -2,10 +2,31 @@
...
-    <br>Welcome to da telephone giphy game, you are on index.
-    <img src="{% static 'img/giphy_logo_laser_black.gif' %}" alt="">
+<div class="row">
+  <div class="col-md-2"></div>
+  <div class="col-md-8">
+	  <p>TeleGiphy is based off the icebreaker game, “Telephone”, where an original message is passed along a chain of players through whispers, and is entertaining through the often very distorted message at the end.<br>TeleGiphy is a spin-off of the game that relies on Giphy’s magnificent random-ness when selecting gifs instead of whispers to distort messages.
+	  </p>
+	  <br>
+	  <!--<p>Powered by <a href="http://giphy.com/"><img src="{% static 'img/giphy_logo_laser_black.gif' %}" alt=""></a></p>-->
+	  <p><a href="/game/{{ token }}"><button type="button" class="btn btn-primary btn-lg btn-block">New game</button></a></p>
```
Comment (Mester) [inline tele_giphy/game/templates/game/index.html]: Using the {% url %} tag would be a better idea https://docs.djangoproject.com/en/1.10/ref/templates/builtins/#url

File: tele_giphy/game/templates/game/index.html [hunk 2]
```diff
@@ -2,10 +2,31 @@
...
+		<div class="input-group-addon">Game token</div>
+		<input type="text" class="form-control" id="exampleInputAmount" placeholder="Amount">
+		<a type="button" class="btn btn-info"><div class="input-group-addon">Enter game</div></a>
+    </div>
   </div>
+</form> -->
+
+	  <input type="text" class="form-control" placeholder="e.g. 9000"></p>
+	  <p>
+	  <a href="/game/{{ token }}"><button type="button" class="btn btn-info">Enter game</button></a>
```
Comment (Mester) [inline tele_giphy/game/templates/game/index.html]: Same here

File: tele_giphy/game/templates/game/wait.html [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+			<li>Everyone of Bob's friends also had to do the same thing. So, Bob receives a GIF from another person. Bob has to now describe the GIF he received. Bob then clicks the "GIF ME" button and he can keep on choosing again just like before between different GIFs until 10 seconds pass.
+			<li>This is repeated again until Bob receives the guess for his original GIF. At this point a list will be presented with all the GIFs and descriptions for Bob's initial word.</li>
+		</ul>
+		<center>
+		<p><strong>The token for you're game is:</strong></p>
+		<p>{{ token }}</p>
+		<!-- if it is the creator this button should appear, but for now it'll appear for everyone -->
+		<p><a href="start/"><button type="button" class="btn btn-success">Start Game</button></a></p>
```
Comment (Mester) [inline tele_giphy/game/templates/game/wait.html:21]: Same here

File: tele_giphy/game/views.py [hunk 4]
```diff
@@ -1,7 +1,14 @@
...
+import random
 
 # Create your views here.
 
 def index(request):
-    return render(request, 'game/index.html')
\ No newline at end of file
+	token = ""
+	for i in range(4):
+		token = token + random.choice('1234567890')
```
Comment (Mester) [inline tele_giphy/game/views.py]: This seems a bit strange. You should easily be able to do it without a for-loop

File: tele_giphy/tele_giphy/urls.py [hunk 5]
```diff
@@ -18,6 +18,7 @@
 
 urlpatterns = [
     url(r'^', include('game.urls')),
+    url(r'^game/', include('game.urls')),
```
Comment (Mester) [inline tele_giphy/tele_giphy/urls.py]: Two urls pointing to the same place? What's the plan here? I dunno if it's good or bad, never seen it before though

File: tele_giphy/templates/base.html [hunk 6]
```diff
@@ -26,12 +26,18 @@
 <body>
 {% block body %}
   {% block header %}
-
+  <div class="container">
+    <div class="row">
+      <div class="col-md-4"></div>
+      <div class="col-md-4"><h1 id="tg"><center>TeleGiphy</center></h1></div>
+      <div class="col-md-4"></div>
```
Comment (Mester) [inline tele_giphy/templates/base.html]: You can do this with offset instead of empty divs http://getbootstrap.com/css/#grid-offsetting

**Pull request merged**